dist/realtime.js

```diff
@@ -38,9 +51,10 @@
         pullInterval: 10000,
         pullIntervalEnhance: 60000,
         forceNew: true,
-        inactivityTimeout: 0
+        inactivityTimeout: 0,
```

Comment:
It has already been removed in 0ded68a, I don't know why Github doesn't show it oo?